tests/console/yast2_clone_system.pm

```diff
@@ -70,6 +69,11 @@ sub run {
     select_console 'root-console';
 }
 
+sub get_ftp_uri {
```

Comment:
This was to make it more 'clean'. In this particular situation i wanted to avoid to do multiple if to get the wanted outcome. i have to admit that your suggestion is also neat, if not better than mine